PowerValidatedSolutions.psm1

```diff
@@ -13749,6 +13776,24 @@ Function Invoke-IomDeployment {
         .EXAMPLE
         Invoke-IomDeployment -jsonFile .\iomDeploySpec.json -certificates ".\certificates\" -binaries ".\binaries\"
         This example configures Intelligent Operations and Management for VMware Cloud Foundation using the JSON spec supplied
+
+        .PARAMETER jsonFile
+        The fully qualified path to the JSON specification file (.json).
+
+        .PARAMETER certificates
+        The fully qualified path to the certificates directory.
+
+        .PARAMETER binaries
+        The fully qualified path to the binaries directory.
+
+        .PARAMETER useContentLibrary
+        Use a vSphere Content Library to store the binaries.
+
+        .PARAMETER contentLibrary
+        The name of the vSphere Content Library to store the binaries.
+
```

Comment:
This line can be removed, just noticed that this line can also be removed as the Content Library name comes from the JSON Spec:

`[Parameter (Mandatory = $false, ParameterSetName = 'useContentLibrary')] [ValidateNotNullOrEmpty()] [String]$contentLibrary,`

Reply:
Removed in e9dfab6 along with the related .PARAMETER item.